samples/browser/virtualdom/virtualdom.fsx

```diff
@@ -56,16 +56,16 @@ type CounterAction =
     | Decrement of int
     | Increment of int
 
-let fakeAjaxCall model h = 
+let fakeAjaxCall model (h:CounterAction->unit) = 
     let message = if model < 30 then Increment 10 else Decrement 5 
     if model > 30 && model < 60 then () 
     else window.setTimeout((fun _ -> h (message)), 2000) |> ignore
 
-let counterUpdate model action =
-    match action with
+let counterUpdate model command =
+    match command with
     | Decrement x -> model - x
     | Increment x -> model + x
-    |> (fun m -> m,[],[fakeAjaxCall model])
+    |> (fun m -> m, (fakeAjaxCall model) |> action) 
```

Comment:
If I understand well, this `action` function is a wrapper to transform the left into a list of action.

I would change the named of the function to something like: `toActionList`
Just to understand what we are doing with this function.